src/IO/GEHDF5Wrapper.cxx

```diff
@@ -246,10 +268,12 @@ Succeeded GEHDF5Wrapper::initialise_listmode_data(const std::string &path)
 {
     if(path.size() == 0)
     {
-        if(is_signa)
+        if(rdf_ver==9) 
         {
             m_address = "/ListData/listData";
             //! \todo Get these numbers from the HDF5 file
+            // AB: todo my file does not have anything in this addres. 
+            // AB only *.BLF file has this address, but not the below variables.
```

Comment:
I'm confused about these comments. Yes, only BLF file with have this address. what do you mean with "below variables"?

Reply:
Literally info about the variables below i.e. `m_size_of_record_signature = 6; m_max_size_of_record = 16;`. In MATLAB, If I open ` "/ListData/listData"` I get a list. When I check its contents  get:

```
>> info.Groups(9).Datasets

ans = 

  struct with fields:

          Name: 'listData'
      Datatype: [1×1 struct]
     Dataspace: [1×1 struct]
     ChunkSize: 33554432
     FillValue: 0
       Filters: []
    Attributes: []

>> info.Groups(9).Datasets.Datatype

ans = 

  struct with fields:

          Name: ''
         Class: 'H5T_INTEGER'
          Type: 'H5T_STD_U8LE'
          Size: 1
    Attributes: []

>> info.Groups(9).Datasets.Dataspace

ans = 

  struct with fields:

       Size: 31310208
    MaxSize: Inf
       Type: 'simple'
```

None of this translates obviously to the variables we need to replace in that function. They may be somewhere else, haven't investigated enough of it yet.

Reply:
This numbers are not recorded in the HDF5 file (at least not for RDF9). They are part of the (previously confidential) info that we've received from GE. So, leave the code as-is, or add a one liner explanation "info provided by GE" or something.

Reply:
@KrisThielemans  sounds good! There was a `\\todo read from file` there, so it got me confused. Will add a comment.